doc/source/web.rst

```diff
@@ -1,7 +1,44 @@
 .. _web:
 
-Web
-===
+Streaming data from cloud sources
+=================================
+
+Spatial Temporal Asset Catalogs
+-------------------------------
+
```

Comment:
Geowombat integrates easy access to Spatial Temporal Asset Catalog (`STAC <https://stacspec.org/en>`_) APIs. STAC is a standardized way to expose collections of spatial temporal data for easy data retrieval of products like Sentinel-2, Landsat, Digital Earth, and even Google Earth Engine products. For a full list of public STAC APIs refer `to the following list<https://stacspec.org/en/about/datasets/>`_.  

Geowombat `open_stack` currently supports:  (**MAYBE INSERT BETTER WAY TO LIST AVAILABLE STACKS/PRODUCTS**) 
```
element84 = 'https://earth-search.aws.element84.com/v0'
google = 'https://earthengine-stac.storage.googleapis.com/catalog/catalog.json'
microsoft = 'https://planetarycomputer.microsoft.com/api/stac/v1'
```

Reply:
For each stack is there a certain set of supported products? 

Also I think people might be interested in using the google earth engine catalog, for `collection` would they just insert the image collection name from GEE? like     `"ASTER/AST_L1T_003"`

Reply:
Unfortunately, the Earth Engine STAC catalog isn't really formatted to work with `pystac`. We can keep our eye on it, but I couldn't get it to work.

Reply:
Ok good to know, should we comment it out for now?